plugins/server/plugin.go

```diff
@@ -131,8 +131,8 @@ func (server *Plugin) NewWorkerPool(ctx context.Context, opt poolImpl.Config, en
 		return nil, errors.E(op, err)
 	}
 
-	list := make([]events.Listener, 0, len(listeners))
-	list = append(list, server.collectPoolLogs)
+	list := make([]events.Listener, 0, len(listeners)+1)
```

Comment:
Please, correct this fix without innefectual assignments:
```go
    list := make([]events.Listener, 0, 1)
    list = append(list, server.collectPoolLogs)
    if len(listeners) != 0 {
        list = append(list, listeners...)
    }
```
And also, add the description in the ticket.